### PR TITLE
Enforce coding style rules (issue #9)

### PR DESCRIPTION
**Pull request by dbenchi**
# New rules:

> require newline after variable declaration
> require newline before return
> require newline after use strict
> require curly braces for: "if", "else", "for", "while", "do", "try", "catch", "switch"
> prevent trailing whitespaces
> prevent unused variables
> validate indentation (\* 2 spaces)

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-	"requireTrailingComma": null,
-	"disallowMultipleVarDecl": null,
-	"requirePaddingNewLinesBeforeLineComments": null,
-	"safeContextKeyword": ["self"],
-	"requireCamelCaseOrUpperCaseIdentifiers": null
+  "preset": "airbnb",
+  "requirePaddingNewLinesAfterBlocks": null,
+  "requirePaddingNewLineAfterVariableDeclaration": true,
+  "requirePaddingNewlinesBeforeKeywords": ["return"],
+  "requireCurlyBraces": ["if", "else", "for", "while", "do", "try", "catch", "case", "default"],
```
Comment (rpignolet) [inline .jscsrc]: `switch` is missing.

Comment (dbenchi) [inline .jscsrc]: Done

**Comment by thomas-hilaire** [comment]
:+1:

**Comment by rpignolet** [comment]
:+1:

**Comment by thomas-hilaire** [comment]
Replaced by https://github.com/linagora/jmap-client/pull/16

**Pull request closed without merging**